android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/Physical.kt

```diff
@@ -55,13 +92,21 @@ class Physical {
             .putExtra(CameraScanActivity.BUNDLE_EXTRA, bundle)
     }
 
+    /**
+     * Handles the scanning result and resolves or rejects the plugin call accordingly.
```

Comment:
Improve the description for someone that doesn't know how the capacitor sdk works.